openslides/core/config.py

```diff
@@ -19,32 +19,34 @@ class ConfigHandler:
     object. To get a config variable use x = config[...], to set it use
     config[...] = x.
     """
+
+    def __init__(self):
+        # Dict, that keeps all ConfigVariable objects. Has to be set at statup.
+        # See the run method in openslides.core.apps.
+        self.config_variables = {}
+
     def __getitem__(self, key):
         """
-        Returns the value of the config variable. Builds the cache if it
-        does not exist.
+        Returns the value of the config variable. Returns the default value, if
+        not value exists in the database.
         """
         try:
```

Comment:
I would like to keep my implementation. It raises an ConfigNotFound for the case that an unknown ConfigVariable is requests that exists in the database. This could happen if a ConfigVariable is deleted in the developmentprocess